decidim-elections/config/locales/en.yml

```
@@ -23,10 +30,24 @@ en:
         title: Actions
         view_deleted_elections: View deleted elections
       admin:
+        census:
+          edit:
+            choose_census: Choose the type of census you want to use for this election
+            created_at: Created at
+            identifier: User identifier
+            title: Type of census
+            voters_count: There are currently %{count} people eligible for voting in this election (this might change on a dynamic census).
```

In romanian we use : 
none: **Nu** exista persoan**e** .... or **Nu** exista nicio persoan**a**
one: Exista 1 persoan**a** 
other: Exista 7 persoan**e** 


```suggestion
            voters_count: 
              none: There is currently no person eligible for voting in this election (this might change on a dynamic census).
              one: There is currently %{count} person eligible for voting in this election (this might change on a dynamic census).
              other: There are currently %{count} people eligible for voting in this election (this might change on a dynamic census).
```